packages/flutter/test/material/segmented_button_test.dart

```diff
@@ -304,8 +304,8 @@ testWidgets('SegmentedButton shows checkboxes for selected segments', (WidgetTes
 
     Finder textHasIcon(String text, IconData icon) {
       return find.descendant(
-        of: find.widgetWithText(Row, text),
-        matching: find.byIcon(icon)
+          of: find.widgetWithText(Row, text),
```

Comment:
This indent isn't needed. LIkewise for lines 308, 344, 345. I'll merge a fix.
```suggestion
        of: find.widgetWithText(Row, text),
```